src/Illuminate/Auth/EloquentUserProvider.php

```diff
@@ -154,6 +154,20 @@ public function createModel()
         return new $class;
     }
 
+    /**
+     * Get a new query builder for the model instance.
+     *
```

Comment:
Add your param to doc_block?

Reply:
thanks. Changed in new commit https://github.com/laravel/framework/pull/27734/commits/b896cce9b4494dd776c5124387603cb109d32b59